### PR TITLE
OCRAD-20 :- add concept map type document.

### PR DESCRIPTION
**Pull request by Akayeshmantha**

### CODE REVIEW

File: Concepts/concept_map_type.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+* Repeat the steps and create the concepts PLANNING PREGNANCY and CURRENTLY PREGNANT of Class Finding and Datatype Boolean. 
+The last possible answer will be OTHER of Class Misc and Datatype N/A. After creating three new concepts, you can edit 
+ANTENATAL VISIT REASON and add them as answers.
+
+## Available operations. 
+
+1. [List concept_map types](#list-concept-map-types)
+2. [Create a concept_map_type](#create-a-concept-map-type)
+3. [Update a concept_map type](#update-a-concept-map-type)
+4. [Delete a concept_map type](#delete-a-concept-map-type)
```
Comment (bmamlin) [inline Concepts/concept_map_type.md]: * List concept mapping types
* Create a concept mapping type
* Update a concept mapping type
* Delete a concept mapping type

File: Concepts/concept_map_type.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+* #### List all non-retired concept map types.
+
+    Quickly filter concept map types with a given search query.Returns a `404 Not Found` status if concept map type not exists. 
+    If user not logged in to perform this action,a `401 Unauthorized` status returned.
+
+    ##### Query Parameters
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *q* | `Search Query` | Display Name of concept map Type.
```
Comment (bmamlin) [inline Concepts/concept_map_type.md]: Type is "String"

Description is "Full or partial name search term. Search is case insensitive."

File: Concepts/concept_map_type.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+    ```console
+    GET /conceptmaptype?q="Search Query"
```
Comment (bmamlin) [inline Concepts/concept_map_type.md]: Space is not allowed in a URL. Perhaps something like:

`GET /conceptmaptype?q=same`

File: Concepts/concept_map_type.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+* To Create a concept map type you need to specify below attributes in the request body. If you are not logged in to perform this action,
+ a `401 Unauthorized` status returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | `String` | Name of the concept attribute type (Required)
+    *description* | `String` | Description
```
Comment (bmamlin) [inline Concepts/concept_map_type.md]: "A brief description of the concept mapping type"

File: Concepts/concept_map_type.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+### Create a concept map type
+
+* To Create a concept map type you need to specify below attributes in the request body. If you are not logged in to perform this action,
+ a `401 Unauthorized` status returned.
+
+    #### Attributes
+
+    Parameter | Type | Description
+    --- | --- | ---
+    *name* | `String` | Name of the concept attribute type (Required)
```
Comment (bmamlin) [inline Concepts/concept_map_type.md]: I think you mean "concept mapping type" (not "concept attribute type")

**Review by bmamlin — CHANGES_REQUESTED** [review]

**Comment by Akayeshmantha** [comment]
@bmamlin  did the review fixes here

**Pull request merged**